Bearded.Graphics/Windowing/Window.cs

```diff
@@ -44,64 +42,60 @@ public void AttachContextToCallingThread()
             }
         }
 
-        private TaskCompletionSource<Void>? initialization;
-
-        private NativeWindowWrapper? window;
+        private NativeWindowWrapper window = null!;
```

Comment:
Why bypass the null safety check here using `null!` if we can just handle it using a much more descriptive message as we did in the previous version. Sure, you shouldn't access `Window` before you call `Run()`, but now if you do that you get a null reference exception and have to dig into our source to find out you have to call `Run()` first. Instead, we could just provide our own custom error message to document that.

Reply:
I figure the whole thing still needs redoing, so didn't bother too much with edge cases.